src/main/docker/configs/logback.xml

```diff
@@ -9,7 +9,7 @@
   </appender>
 
   <appender name="FILE" class="ch.qos.logback.core.FileAppender">
-    <file>/var/log/hauth/hauth-${byDay}.log</file>
```

Comment:
Would probably be good to make this change to [the template project](https://github.com/ksclarke/vertx-template/blob/e25163915e33d8dcca9e6ce967903254fcebf1de/src/main/docker/configs/logback.xml#L12) as well; I wasn't expecting Hauth to be referenced there, so didn't catch it in my search-and-replace.